fs/api_internal.go

```diff
@@ -145,9 +145,9 @@ func (vS *volumeStruct) inFlightFileInodeDataFlusher(inodeNumber inode.InodeNumb
 
 	// Act as if a package fs client called Flush()...
 
-	inodeLock, err = vS.initInodeLock(inodeNumber, nil)
+	inodeLock, err = inode.InitInodeLock(vS.volumeName, inodeNumber, nil)
```

Comment:
Ok... now that I could see the formerly collapsed diff of fs/api_internal.go, I'll codify my preference here... Instead of ```inode.InitInodeLock(vs.volumeName, inodeNumber, nil)```, I would have expected to see ```vS.inodeVolumeHandle.InitInodeLock(inodeNumber, nil)```. See the difference?

Reply:
Added a receiver for the functions related to wrapping DLM locks for inodes.